cds/modules/migrator/records.py

```diff
@@ -645,7 +646,11 @@ def _resolve_user_id(cls, email=None):
         logging.debug('Set correct user id from email {0}.'.format(email))
         if not email:
             return -1
-        return User.query.filter_by(email=email.lower()).one().id
+        try:
+            return User.query.filter_by(email=email.lower()).one().id
+        except NoResultFound:
+            logging.error('Email not found {0}'.format(email))
+            return -1
```

Comment:
I think it's better to return the id of `cds.support` instead, probably it will avoid some funny situations in the future

Reply:
@egabancho done